requirements.txt

```diff
@@ -7,3 +7,7 @@ dwave-hybrid==0.6.13
 
 # Development requirements
 reno==4.1.0
+
+# Examples requirements
+torchvision==0.22.0
```

Comment:
this should probably go in a separate `requirements-examples.txt` (?) @thisac

Reply:
Since we are not uploading any example for now, I removed these requirements.